src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int numberBall = 100;
```

Comment:
it should be constant

Reply:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants